repo/packages/N/nexus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "1.0-3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "support@sonatype.org",
+  "website": "http://www.sonatype.org/nexus/",
+  "framework": false,
+  "description": "Nexus is a software repository manager. It supports Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories",
+  "tags": ["artifact", "maven", "npm", "package", "repository", "bower", "nuget", "pypi"],
+  "preInstallNotes": "Having Marathon-lb installed and setting a virtual host in the config is required for being able to access Nexus. If you're using a host volume, make sure that the folder exists on the host you used for host pinning, and that you did a 'chown -R 200:200' on the folder. Default credentials are admin/admin123. Please change them after the first login.",
```

Comment:
Will do. I think it makes sense to mention this in the https://github.com/mesosphere/universe#submit-your-package paragraph as well.